backend/src/main/java/com/tradomator/model/Wallet.java

```diff
@@ -0,0 +1,20 @@
+package com.tradomator.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Wallet {
```

Comment:
Could probably also be a record.

Reply:
yes, thanks. same here. As there appear potential uncontrollable failures , i would keep for now and remember next time.